### PR TITLE
REP-1246 Update XSD namespace

### PR DESCRIPTION
**Pull request by wdschei**
**OLD Namspace:**
   `docs.api.rackspacecloud.com/repose/`
   `docs.rackspacecloud.com/repose/`
   `openrepose.org/repose/`
   `openrepose.org/components/`

**NEW Namspace:**
   `docs.openrepose.org/repose/`

### CODE REVIEW

**Comment by wdschei** [comment]
NOTE: This is not the backwards compatible part yet, rather it is just the updating of all the existing configuration files both actual and in code.

**Comment by wdschei** [comment]
This should be mergeable again after I rebased it and the tests finish up.

File: repose-aggregator/commons/configuration/src/main/java/org/openrepose/commons/config/parser/jaxb/UnmarshallerValidator.java [hunk 1]
```diff
@@ -55,11 +79,80 @@ public Object validateUnmarshal( InputStream inputstream ) throws JAXBException,
...
+                                if (attribute != null) {
+                                    String namespace = attribute.getNodeValue();
+                                    if (namespace != null) {
+                                        if (namespace.contains(OLD_NAMESPACES[0]) ||
+                                                namespace.contains(OLD_NAMESPACES[1]) ||
+                                                namespace.contains(OLD_NAMESPACES[2]) ||
+                                                namespace.contains(OLD_NAMESPACES[3]) ||
+                                                namespace.contains(OLD_NAMESPACES[4]) ||
+                                                namespace.contains(OLD_NAMESPACES[5])) {
+                                            LOG.error("Using old namespace - {}", namespace);
```
Comment (adrianjgeorge) [inline repose-aggregator/commons/configuration/src/main/java/org/openrepose/commons/config/parser/jaxb/UnmarshallerValidator.java]: Make it a warn, and tell them what the new name space is

**Comment by adrianjgeorge** [comment]
Should add a time bomb into the test so that we come back in and remove the hack. Set it for 6 months since that is the next antiicpated major release

**Comment by wdschei** [comment]
A time bomb for September w/ comment was added to the test.

**Comment by james49** [comment]
Retest this please.

**Pull request merged**